From 9868ab358dce27ed969e24c6865688e60c9e15c6 Mon Sep 17 00:00:00 2001
From: Bai Ping <b51503@freescale.com>
Date: Wed, 3 Jun 2015 22:24:41 +0800
Subject: [PATCH 1174/1594] MLK-11021-01 ARM: dts: imx6: correct ocram and gpt
 node property of imx6ul

commit d807a761bc91c12aff144711d6db09c898e9acc0 from
git://git.freescale.com/imx/linux-2.6-imx.git

* The ocram used for busfreq is at 0x904000.

* Update the gpt clocks properties.

Signed-off-by: Bai Ping <b51503@freescale.com>
---
 arch/arm/boot/dts/imx6ul.dtsi |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ul.dtsi b/arch/arm/boot/dts/imx6ul.dtsi
index 00b66cf..d3fe5ef 100644
--- a/arch/arm/boot/dts/imx6ul.dtsi
+++ b/arch/arm/boot/dts/imx6ul.dtsi
@@ -141,9 +141,9 @@
 			reg = <0x00900000 0x4000>;
 		};
 
-		ocrams_ddr: sram@00901000 {
+		ocrams_ddr: sram@00904000 {
 			compatible = "fsl,ddr-lpm-sram";
-			reg = <0x00901000 0x1000>;
+			reg = <0x00904000 0x1000>;
 		};
 
 		ocram: sram@00905000 {
@@ -424,8 +424,8 @@
 				compatible = "fsl,imx6ul-gpt", "fsl,imx31-gpt";
 				reg = <0x02098000 0x4000>;
 				interrupts = <GIC_SPI 55 IRQ_TYPE_LEVEL_HIGH>;
-				clocks = <&clks IMX6UL_CLK_DUMMY>,
-					 <&clks IMX6UL_CLK_DUMMY>;
+				clocks = <&clks IMX6UL_CLK_GPT1_BUS>,
+					 <&clks IMX6UL_CLK_GPT1_SERIAL>;
 				clock-names = "ipg", "per";
 			};
 
-- 
1.7.5.4

